reference/pipeline/stages/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: single
+title:  "Pipeline Stages"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+This article describes the currently-supported stages that you can add to your
+Spinnaker pipelines.
+
+## General
+
+### Bake
+Bakes an image in the specified region.
+
+### Canary Analysis
+Runs a canary task.
```

Comment:
Runs automated canary analysis against the deployment before fully deploying.

For this one, and probably for others, you can link to more information. In this case /guides/user/canary/